manifests/event_management.pp

```diff
@@ -0,0 +1,19 @@
+# TODO: Properly document this class
+class servicenow_reporting_integration::event_management (
+  String[1] $instance,
+  Optional[String[1]] $user                                                                  = undef,
+  Optional[String[1]] $password                                                              = undef,
+  Optional[String[1]] $oauth_token                                                           = undef,
+  Optional[String[1]] $pe_console_url                                                        = undef,
+  String $servicenow_credentials_validation_table                                            = 'em_event',
+) {
+  class { 'servicenow_reporting_integration':
+    operation_mode                          => 'event_management',
```

Comment:
ah cool, so they won't even be allowed to set any of the optional incident params if using the event management class.

Reply:
Yup, exactly. That's why we went w/ the split.